resources/js/index.js

```diff
@@ -0,0 +1,49 @@
+$(function() {
+  $('.feedback-slide').slick({
+    dots: true,
+    infinite: false,
+    speed: 300,
+    slidesToShow: 2,
+    slidesToScroll: 1,
+    prevArrow: '<button class="slide-arrow prev-arrow"><i class="fas fa-angle-left"></i></button>',
+    nextArrow: '<button class="slide-arrow next-arrow"><i class="fas fa-angle-right"></i></button>',
+    responsive: [
+      {
+        breakpoint: 1024,
+        settings: {
+          slidesToShow: 1,
+          slidesToScroll: 1,
+          infinite: true,
+          dots: true
+        }
+      },
+      {
+        breakpoint: 600,
+        settings: {
+          slidesToShow: 1,
+          slidesToScroll: 1
+        }
+      },
+    ]
+  });
+
+  $(".footer-iconandlink").mouseover(function() {
+    $( "#imgContact", this ).removeClass("txt-hidden").addClass( "readmore");
+  });
+
+  $(".footer-iconandlink").mouseout(function() {
+    $( "#imgContact", this ).removeClass("readmore").addClass("txt-hidden");
```

Comment:
thử tìm hiểu 
```
$( "#imgContact", this )
```
khác gì 
```
$( "#imgContact")
```

Reply:
this này có nghĩa là khi con chuột trỏ tới đâu thì nó sẽ tác động ở đó
theo em nghĩ là 2 cái này nó không khác nhau